### ISSUE TITLE
Support cross-posted (embedded) Reddit posts

### ISSUE THREAD

**Issue by Zegnat** [spikecodes/libreddit#13]
**Steps to reproduce:**

1. Find a post on one subreddit wholesale containing a post from a different subreddit.
2. Visit it on libreddit.

**Result:** the entire post only includes a link out to the original post, which will also take you outside of libreddit and back onto Reddit-proper.

Example URL: `https://libredd.it/r/offlineTV/comments/klq0ji/crackhead_on_the_loose/`.

**Expected:** see the post as it is on the original subreddit it was crossposted from.

In the case of the above example URL, I would expect the same content as displayed for the URL: `https://libredd.it/r/valkyrae/comments/klhie9/rock_mannn/`.

**Comment by spikecodes** [comment]
Thanks for the bug report! I've added this to my to-do list.

**Comment by se7enge** [comment]
Yes, this is the only major thing that I miss tbh, as it makes you have to open a new link in order to see the crossposted content.

E.g. `https://libredd.it/r/commandline/comments/vfxfe3/psparklines_is_now_available_on_the_psgallery/` shows the link to the original post without displaying its content, whereas on reddit proper `https://www.reddit.com/r/commandline/comments/vfxfe3/psparklines_is_now_available_on_the_psgallery/` it shows the post embedded, which is much more convenient.
